packages/date/src/index.js

```diff
@@ -119,6 +119,12 @@ function setupWPTimezone() {
 // the attached timezone, instead of setting a default timezone on
 // the global moment object.
 export const moment = ( ...args ) => {
+	deprecated( 'wp.date.moment', {
+		version: '4.4',
+		alternative: 'the moment script as a dependency',
```

Comment:
This is asking developers to include `moment` as their own, explicit dependency, right? Would

```suggestion
		alternative: 'include moment as a dependency in your code',
```

be better? Though that could lead to a fair few blocks including `moment` in their builds 😢

Reply:
I wrote it this way because of the way `deprecated` generates the message. In your case it will return: "use include moment as a dependency in your code instead"

Reply:
Ah, I figured but I forgot how it was output. 👍